src/reports/components/report-sections/passed-checks-section.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+.passed-checks-container {
+    padding-bottom: 58px;
+
+    :global {
+        .title-container[aria-level='2'] {
```

Comment:
@jalkire , FYI, your heading level PR will likely have a silent/semantic merge conflict with this